lib/health_cards/health_card.rb

```diff
@@ -138,7 +138,7 @@ def types
       # Check if this class supports the given type claim(s)
       # @param type [Array, String] A type as defined by the SMART Health Cards framework
       # @return [Boolean] Whether or not the type param is included in the types supported by the HealthCard (sub)class
-      def supports_type?(*type)
+      def supports_type?(type)
```

Comment:
The documentation says this method can take a string, but it looks to me like it will only work with an array.

Reply:
Fix code so that it accepts either